### PR TITLE
SPM-94 Add Linter to backend

### PR DESCRIPTION
**Pull request by brandontjd**
# Summary

We can easily forget double imports or unhygienic code at times when we are rushing out features. Having a linter will help catch our blindspots. Template is taken from [here](https://medium.com/swlh/enhancing-code-quality-with-github-actions-67561c6f7063). We can relax the linter if we need to. Should expect to see many fail cases when first implemented.

### CODE REVIEW

**Comment by brandontjd** [comment]
Linter is working fine, we have a lot of technical "debt"in terms of hygiene to clean up.

**Review by joelytj — APPROVED** [review]

File: .github/workflows/python-linter.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6.9
```
Comment (yarkhinephyo) [inline .github/workflows/python-linter.yml:15]: Should it be diff version?

**Review by brandontjd — COMMENTED** [review]
Very neat, thanks @yarkhinephyo !

**Comment by brandontjd** [comment]
So just for the record, before we commit , remember to run two things

1. `pytest`
2. `sh run_linter.sh`

@joelytj

**Pull request merged**